docs/src/modules/java-protobuf/pages/actions-as-controller.adoc

```diff
@@ -59,14 +59,13 @@ Java::
 ----
 include::example$java-protobuf-valueentity-shopping-cart/src/main/java/com/example/shoppingcart/ShoppingCartActionImpl.java[tag=initialize]
 ----
-<1> We generate a new UUID
-<2> We use `components().shoppingCart().create(...)` to create a `DeferredCall` for `create` on the shopping cart
-<3> `execute()` on the `DeferredCall` immediately triggers a call and returns a `CompletionStage` for the response
-<4> Once the call succeeds or fails the `CompletionStage` is completed or failed, we can transform the result from `CompletionStage<Empty>`
-    to `CompletionStage<Effect<NewCartCreated>>` using `handle`.
-<5> On a successful response, we create a reply effect with a `NewCartCreated`
-<6> If the call leads to an error, we create an error effect asking the client to retry
-<7> `effects().asyncEffect()` allows us to reply with a `CompletionStage<Effect<NewCartCreated>>`
+<1> We generate a new UUID.
+<2> We use `components().shoppingCart().create(...)` to create a `DeferredCall` for `create` on the shopping cart.
+<3> `execute()` on the `DeferredCall` immediately triggers a call and returns a `CompletionStage` for the response.
+<4> Once the call succeeds or fails the `CompletionStage` is completed or failed, we can transform the result from `CompletionStage<Empty>` to `CompletionStage<Effect<NewCartCreated>>` using `handle`.
+<5> On a successful response, we create a reply effect with a `NewCartCreated`.
+<6> If the call leads to an error, we create an error effect asking the client to retry.
+<7> `effects().asyncEffect()` allows us to reply with a `CompletionStage<Effect<NewCartCreated>>`.
```

Comment:
Change is not visible here because it's only the removal of a breakline. I was using a regex to find lines not ending with a `.`. Removing unnecessary breaklines helped me exclude those false positives.